pages/impressum.vue

```diff
@@ -0,0 +1,157 @@
+<script setup></script>
+
+<template>
+    <div class="top-nav">
+        <NuxtLink to="/">
+            <img src="/public/TextwithBg.svg" alt="" class="top-nav__logo" />
+        </NuxtLink>
+    </div>
+
+    <div class="container-fluid">
+        <div class="impressum">
+            <div class="content">
+                <h1>Imprint</h1>
+                <p>
+                    comixcraft is a project developed at the St. Pölten University of Applied Sciences (FHSTP), Austria,
+                    as part of the MediaLab semester project of the Creative Computing program. comixcraft is based on
+                    <NuxtLink class="linkText" to="https://fhstp.github.io/comixplain/" target="_blank"
+                        >Comixplain
+                    </NuxtLink>
+                    and utilizes the existing assets from that project.
+                </p>
+            </div>
+
+            <div class="content">
+                <h2>Collected Data</h2>
+                <p>
+                    The application does not collect any personal data. The only data collected is the user's input for
+                    the comic creation process, which is stored in the browser's local storage. This data is not shared
+                    with any third party.
+                </p>
+            </div>
+
+            <div class="content">
+                <h2>Contributors</h2>
+                <ul class="lists">
+                    <li>Anna Blasinger</li>
+                    <li>Florent Gabillon</li>
+                    <li>Alica Holla</li>
+                    <li>Vanessa Köck</li>
+                    <li>Juliana Matsumura</li>
+                    <li>Antoine Muneret</li>
+                    <li>Hannah Wagner</li>
```

Comment:
Personally, I would not do that, since we agreed on the roles being more of a "Person responsible for that area" and it could suggest to the outside reader, that this person was ONLY doing this.
Also, we are not adding roles on the Showcase page as well.